app/src/main/java/com/tehmou/rxbookapp/RxBookApp.java

```diff
@@ -18,7 +18,7 @@
     private Graph mGraph;
 
     @Inject
-    Instrumentation instrumentation;
+    ApplicationInstrumentation mApplicationInstrumentation;
```

Comment:
Is there a reason for the 'm' here? As far as I know it's a platform convention for class members, though with Android Studio color coding it is considered redundant.